app/assets/javascripts/govuk_publishing_components/vendor/lux/lux-reporter.js

```diff
@@ -654,14 +780,13 @@
         hCPUDetails[jsType] = "";
       }
       var hStats = cpuStats(hCPUDetails[jsType]);
-      var sStats =
-        ",n|" +
-        hStats["count"] +
-        ",d|" +
-        hStats["median"] +
-        ",x|" +
-        hStats["max"] +
-        (0 === hStats["fci"] ? "" : ",i|" + hStats["fci"]); // only add FCI if it is non-zero
+      var sStats = ",n|" +
+      hStats["count"] +
+      ",d|" +
+      hStats["median"] +
+      ",x|" +
+      hStats["max"] +
+      (0 === hStats["fci"] ? "" : ",i|" + hStats["fci"]); // only add FCI if it is non-zero
```

Comment:
Would this indenting change remove the difference?

```suggestion
      var sStats =
        ",n|" +
        hStats["count"] +
        ",d|" +
        hStats["median"] +
        ",x|" +
        hStats["max"] +
        (0 === hStats["fci"] ? "" : ",i|" + hStats["fci"]); // only add FCI if it is non-zero
```

Reply:
Good spot, thanks, have corrected.